internal/pkg/custom/custom_check.go

```diff
@@ -107,7 +107,7 @@ type MatchSpec struct {
 	Name               string      `json:"name,omitempty" yaml:"name,omitempty"`
 	MatchValue         interface{} `json:"value,omitempty" yaml:"value,omitempty"`
 	Action             CheckAction `json:"action,omitempty" yaml:"action,omitempty"`
-	PreConditions      []MatchSpec `json:"preconditions,omitempty" yaml:"preconditions,omitempty"`
+	PreConditions      []MatchSpec `json:"preConditions,omitempty" yaml:"preConditions,omitempty"`
```

Comment:
I think this change is going to break existing custom check configurations.

Reply:
I've tested it, apparently, the parser converts the keys to lowercase